.travis.yml

```diff
@@ -86,46 +86,46 @@ script:
   - ./animate-bmp
   - ./audio_player
   - ./background-effects
-  - ./calculator 
+  #- ./calculator   # https://travis-ci.org/qPCR4vir/nana/jobs/140238537#L1159
```

Comment:
> 0.12s$ ./calculator
X Error of failed request:  BadAtom (invalid Atom parameter)
  Major opcode of failed request:  18 (X_ChangeProperty)

It's strange. I tried to rebuild [travis #181](https://travis-ci.org/cnjinhao/nana/builds/140254800) many times, no X error occurs.